### DISCUSSION TITLE
Can a `vcs` have wilcards in its `url` like `path`?

### DISCUSSION THREAD

**Discussion by bilogic** [composer/composer#11278] [Q&A / Support] [answered]
```
"repositories": [
    {
        "type": "path",
        "url": "/var/www/packages/*/*"
    }
]
```

The above works for me. But if I replace `path` with `vcs`, it throws an exception `No driver found to handle VCS repository /var/www/packages/*/*`.

Is is possible to use wildcards for VCS?

**Comment by stof** [comment, marked as answer]
no it is not. The VCS repository does not support globs to find its URL (as it passes that URL to the VCS driver to clone the repo)

**Reply by bilogic** [reply]
ok, i need to find another way, thanks
